Pod/Classes/JSSAlertView.swift

```diff
@@ -181,19 +181,19 @@ public class JSSAlertView: UIViewController {
 		let titleString = titleLabel.text! as NSString
 		let titleAttr = [NSFontAttributeName:titleLabel.font]
 		let titleSize = CGSize(width: contentWidth, height: 90)
-		let titleRect = titleString.boundingRectWithSize(titleSize, options: NSStringDrawingOptions.UsesLineFragmentOrigin, attributes: titleAttr, context: nil)
```

Comment:
Precisely, I also had the same runtime error, but it seems to work perfectly with `let titleAttr = [NSFontAttributeName:titleLabel.font!] as [String: Any]`